sli/Makefile.am

```diff
@@ -18,7 +18,6 @@ noinst_LTLIBRARIES= libsli.la
 
 libsli_la_SOURCES=\
 		aggregatedatum.h\
-		array.h\
```

Comment:
I am surprised the file is not needed, but if it compiles .... It makes me wonder if there are other examples of dead code around.